src/main/java/org/codehaus/mojo/versions/AbstractVersionsUpdaterMojo.java

```diff
@@ -417,10 +421,31 @@ protected abstract void update( ModifiedPomXMLEventReader pom )
      * @return <code>true</code> if the update should be applied.
      * @since 1.0-alpha-1
      */
+    @Deprecated
     protected boolean shouldApplyUpdate( Artifact artifact, String currentVersion, ArtifactVersion updateVersion )
+    {
+        return shouldApplyUpdate(artifact,currentVersion,updateVersion,false);
+    }
+
+    /**
+     * Returns <code>true</code> if the update should be applied.
+     *
+     * @param artifact The artifact.
+     * @param currentVersion The current version of the artifact.
+     * @param updateVersion The proposed new version of the artifact.
+     * @return <code>true</code> if the update should be applied to the pom.
+     * @since 1.0-alpha-1
```

Comment:
not sure about the since :) 
```suggestion
     * @since 2.9
```